src/app/resources/assort/assort.tsx

```diff
@@ -1,64 +1,156 @@
+          <article className="border-l-2 border-navy-400 pl-4">
+            <p className="mb-3 pt-4">
+              The Aid Standards for Sorting (ASSORT) are a set of
+              interoperability standards for humanitarian aid that are designed
+              to save grassroots organisations time, effort, and money.
+            </p>
+            <p>
+              When quality control and sorting are not standardised,
+              inefficiencies arise. These inefficiencies require significant
+              expenditure of resources:
+            </p>
+            <ul className="ml-4 list-disc py-4">
+              <li className="mb-2">
+                Quality Control - the receiver may have to pay to dispose of
+                items that are culturally inappropriate, out of season, or poor
+                in quality.
+              </li>
+              <li className="mb-2">
+                Sorting - 100,000 aid items on a shipment might be sorted one
+                way by the sender and then re-sorted another way by the
+                receiver. Sorting takes time.
+              </li>
+            </ul>
+          </article>
+          <article className="border-l-2 border-navy-400 pl-4">
+            <p className="mb-3">
+              <span className="text-text-blue font-bold">
+                That’s where ASSORT comes in.
+              </span>{" "}
+              ASSORT leads to faster, cheaper, and more responsive aid. By
+              providing a “common language” for aid donations, organisations
+              receiving aid can know ahead of time what is on an aid shipment.
+              When the shipment arrives, they process it much faster than they
+              otherwise could. Indeed, a pilot shipment showed that using ASSORT
+              standards significantly cut down on the receiving organisation’s
+              sorting and processing time. Saving time and volunteer capacity
+              makes a big difference for grassroots aid organisations.
+            </p>
+            <p>The ASSORT Guide includes multiple areas of standardisation:</p>
+            <ul className="ml-4 list-disc py-4">
+              <li className="mb-2">
+                How to quality check donated items meant for distribution.
+              </li>
+              <li className="mb-2">
+                How to sort donations into standard categories.
+              </li>
+              <li className="mb-2">
+                How to size clothing donations with a printable ASSORT sizing
+                chart.
+              </li>
+              <li className="mb-2">
+                How to box and label the sorted donations and ready them for
+                shipment.
+              </li>
+            </ul>
+          </article>
+          <article className="border-l-2 border-navy-400 pl-4 mt-8">
+            <p>
+              ASSORT was established as part of a joint partnership between
+              <span>
+                {" "}
```

Comment:
i think we maybe want to clean up some of these spaces

Reply:
@MatrixFrog sure thing!  What did you have in mind instead?  want to include that in this PR?

Reply:
I think let's do a separate one just to get rid of any unnecessary spaces, and then do this one after.

Reply:
actually after looking closer, most of these spaces are necessary, let's just go forward with this